### PR TITLE
CI: Only publish docs on one of the builds.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

**Comment by danielballan** [comment]
Hm, perhaps not. I am not sure. I have pushed a commit to check....

File: .travis.yml [hunk 1]
```diff
@@ -35,7 +35,10 @@ script:
   - flake8 --max-line-length=115  # Enforce code style (but relax line length limit a bit).
   - set -e
   - make -C docs html  # Build the documentation.
-  # Pubish the documentation to GitHub Pages.
-  - pip install doctr
-  - doctr deploy --built-docs docs/build/html --deploy-repo NSLS-II/NSLS-II.github.io event-model
-  - doctr deploy --built-docs docs/build/html --deploy-repo bluesky/bluesky.github.io event-model
+  - |
+    if [ $PUBLISH_DOCS ] then
```
Comment (mrakitin) [inline .travis.yml]: Thanks. I think this is missing:
```suggestion
    if [ $PUBLISH_DOCS ]; then
```

Comment (mrakitin) [reply .travis.yml]: Sorry I wasn't precise about the location of the semicolons. It's still should be a part of `if ... then`.

**Comment by danielballan** [comment]
Oops, that makes sense. You can see my bash-fu could use some work. :-D Fixed and force-pushed.

**Pull request merged**